hc/settings.py

```diff
@@ -77,11 +78,14 @@
 # Default database engine is SQLite. So one can just check out code,
 # install requirements.txt and do manage.py runserver and it works
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME':   './hc.sqlite',
-    }
+    'default': dj_database_url.config()
 }
+# DATABASES = {
```

Comment:
If this is not being used, delete the code

Reply:
Have worked on it